apps/builder/app/builder/features/style-panel/shared/style-info.ts

```diff
@@ -60,6 +60,15 @@ type SourceProperties = {
   [property in StyleProperty]?: SourceValueInfo;
 };
 
+/*
+ * For some cases we are encouraging to use custom defaults, than
+ * `initial` values provided by browsers. This helps in controlling the behaviour
+ * of such properties
+ */
+const CUSTOM_DEFAULT_VALUES: Partial<Record<StyleProperty, StyleValue>> = {
+  outlineWidth: { value: 0, type: "unit", unit: "px" },
+};
+
```

Comment:
Heya, i took a little different approach here. Since the users are allowed to use `keywords` at any time. It is little tricky to always follow up with `keyword` default checks. So, if a element misses the style from `htmlSource`. The logic checks if there are any custom defaults are defined. And if they are missing from here too, then it fallbacks to the auto-generated. I thought of adding cases while generating `__generated__/properties`. But, that effects the actual sources. So

This helps in arranging all the custom defaults too in one place. Does that looks clear, open to suggestions on improving this.

Reply:
I think @istarkov knows this logic from borders section

Reply:
the actual default value is medium, so we would need to use 1 to approximate that

Reply:
@TrySound what's the best way to handle this and rely on our normalize/preset?

Reply:
same should be for borders

Reply:
@istarkov were you just using hardcoded 1px for border width or did you map it from medium to 1 once you got medium from data?

Reply:
I see borders are in normalize.ts, probably outlines should be too

Reply:
But presets needs to import `normaiize` right. Because `outline` can be applied on every element ? We should add to every element ? 🤔 

https://github.com/webstudio-is/webstudio-builder/blob/main/packages/react-sdk/src/components/body.ws.tsx#L2

Reply:
I would say it should be just like the border, let's wait for @TrySound to reply

Reply:
This is where borders set for every tag
https://github.com/webstudio-is/webstudio-builder/blob/main/packages/react-sdk/src/css/normalize.ts#L40

Reply:
Thanks for the tip, added as preset now 👍